util/customTransforrmationsStore.js

```diff
@@ -21,7 +22,13 @@ async function getTransformationCode(versionId) {
     const response = await fetchWithProxy(
       `${getTransformationURL}?versionId=${versionId}`
     );
-    if (response.status !== 200) {
+
+    if (response.status >= 500) {
```

Comment:
This entire block is repeated thrice, can you check if some reuse-able functions can be written?

Reply:
added